docs/src/main/paradox/java/http/index.md

```diff
@@ -26,6 +26,7 @@ akka-http-testkit
 akka-http-jackson
 : Predefined glue-code for (de)serializing custom types from/to JSON with [jackson](https://github.com/FasterXML/jackson)
 
+Akka HTTP API - @javadoc:[Javadoc](akka.http.javadsl.index)
```

Comment:
To link to the package page for `akka.http.javadsl` this should be `akka.http.javadsl.package-summary`, which would generate http://doc.akka.io/japi/akka-http/current/?akka/http/javadsl/package-summary.html